### DISCUSSION TITLE
Need Discount

### DISCUSSION THREAD

**Discussion by Arayaan05** [aclap-dev/video-downloadhelper#1514] [License & Purchase requests]
### What's your issue?

I hope this email finds you well. My name is Muhammad Furqan, a final-year student of BS Architecture at the University of Engineering and Technology, Lahore, registered under the number 2020-arch-42. My academic endeavors often require access to various protected educational databases that are crucial for the comprehensive research needed in my studies.

Given the substantial nature of the data required for my thesis and other academic projects, I am writing to inquire if there are any student licenses or discounts available that could facilitate my access to these resources. The financial aspect of accessing high-quality, protected data can be quite challenging for students, and any assistance in this regard would greatly enhance our learning and research capabilities.

I am committed to adhering to all necessary guidelines and terms of use for the data accessed, and I am prepared to provide any documentation required to verify my student status and eligibility for such a program.

Thank you for considering my request. I look forward to your favorable response and any guidance you can provide on how I may proceed.

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi, sorry there is no discount.

**Reply by Chpoan** [reply]
Not just for students, but for all of LATAM—if there's no differentiated pricing, that business model simply won't work here. I'm used to buying and paying for my software, like CaptureWiz, antivirus, operating systems, editors, compilers, CMS tools. But if there's no regional pricing for LATAM, then there's nothing to be done.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I kept the E-mail I got from VDH when I bought my license.  That E-mail says I paid $28.50 (US dollars) on 2018/5/18.  That is now over 6 & 1/2 years ago.  So I'm in my 7th year of using that license.  That means I paid a bit over $4 per year for that license.  Next year, I'll be able to say I paid even less.  Each year, the license will have cost me less & less.  You see, the license is for life.  That's not the life of my computer.  That's my life.  I have used the same license on 2 computers so far & 2 operating systems.  Who knows how many that will be before I'm done.

I say all of that in order to completely dismantle any argument you might have that the VDH license is expensive.  It's not.  It's ridiculously cheap.  It is a lifetime license, a one-time outlay of cash.  It covers every computer you have now & every computer you will ever have.  If you buy a license for VDH for Firefox, that license will be good for the rest of your life in Firefox on every computer you own now & every computer you will ever own.  There are separate licenses for VDH for Chrome & VDH for Edge, but they operate the same way, costing less each year you use them.

A student discount is unnecessary.  The license discounts itself the longer you use it.

Now, you talk about protected data.  There is a mechanism called Digital Rights Management.  DRM.  DRM protects copyrighted material.  The intended purpose of DRM is to prevent downloading of such material.  VDH does not download content protected by DRM.  That would be a violation of international copyright law.  So if you think you will be able to use VDH to download content protected by DRM, you need to change your thinking.  VDH will not accomplish that.

VDH downloads videos.  That's what the V stands for.  VDH = Video Download Helper.  That's all VDH does.  It doesn't download PDFs, spreadsheets, images of paintings or sculptures, other things.  It downloads videos.  That's it.  If that fits within your needs, great.

BEFORE YOU BUY A LICENSE, test VDH against the content you think is important to your research.  If VDH does not successfully download what you need, then you have learned you correctly did not buy a license & you can move on to find a tool that does suit your needs.  VDH makes no claims to be able to download all videos anywhere on the web.  It does successfully download a lot of videos.  It's actually quite miraculous that it does work on as many sites as it does.  But it does not work everywhere.  Understand that a VDH license does not unlock any features of the software.  A VDH license also does not make VDH work on sites where unlicensed VDH does not work.  So if VDH fails on content you need, buying a license will not make it succeed.  It will still fail.  So be smart & give the product a test drive before you buy a license.
